Eevee.Sleep.Bot/Modules/SlashCommands/RoleManagementSlashModule.cs

```diff
@@ -0,0 +1,69 @@
+using Discord;
+using Discord.Interactions;
+using JetBrains.Annotations;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Utils;
+
+namespace Eevee.Sleep.Bot.Modules.SlashCommands;
+
+[Group("role", "Commands for managing the roles on a user.")]
+public class RoleManagementSlashModule : InteractionModuleBase<SocketInteractionContext> {
+    [SlashCommand("change", "Select a specific one from the role you have")]
+    [UsedImplicitly]
+    public Task ChangeRoleAsync() {
+        return Context.Interaction.RespondAsync(
+            components: DiscordMessageMaker.MakeRoleSelectButton(
+                hasRoles: DiscordRoleContoller.FindRoleIdsByUserId(Context.User.Id), 
+                guildRoles: Context.Guild.Roles),
+            ephemeral: true
+        );
+    }
+
+    [SlashCommand("remove", "Remove the role ownership of a user")]
+    [RequireOwner]
+    [UsedImplicitly]
+    public async Task RemoveRoleAsync(IUser user, IRole role) {
+        await DiscordRoleContoller.RemoveRoles(user.Id, [role.Id]);
+
+        await Context.Interaction.RespondAsync(
+            "Role record deleted.",
```

Comment:
Update the return to show exactly what role is deleted from who, see how to mention role and users by searching `MentionUtils`.

No need `ephemeral` for this.

Reply:
I made it a good message.
[0fb77fb](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/0fb77fb7840dd9fe43cfcdc15bc7f2c9399792ab)